cli/src/cmd/forge/doc.rs

```diff
@@ -24,6 +34,7 @@ impl Cmd for DocArgs {
         DocBuilder::from_config(DocConfig {
             root: self.root.as_ref().unwrap_or(&find_project_root_path()?).to_path_buf(),
             sources: config.project_paths().sources,
+            out: self.out.as_ref().unwrap_or(&PathBuf::from("docs")).to_path_buf(),
```

Comment:
clippy will complain here, should be `self.out.clone().unwrap_or_else(|| PathBuf::from("docs"))`

Reply:
thank you, I am still very new with Rust. Big thanks to @rkrasiuk for helping me with these PRs!